parser.go

```diff
@@ -128,6 +128,9 @@ var hexRE = regexp.MustCompile(`(?i)(\|(?:\s*[a-f0-9]{2}\s*)+\|)`)
 // escapeRE matches char that needs to escaped in regexp.
 var escapeRE = regexp.MustCompile(`([()+.'\\])`)
 
+// metaSplitRE matches string in metadata
+var metaSplitRE = regexp.MustCompile(`,\s*`) //global
```

Comment:
This inline comment isn't necessary (sorry, I put it in my comment to show that it should be a global)